packages/x-date-pickers/src/AdapterDayjs/AdapterDayjs.test.tsx

```diff
@@ -83,7 +83,7 @@ describe('<AdapterDayjs />', () => {
 
     it('should not create UTC dates when no instance passed', () => {
       const onChange = spy();
-      render(<DateCalendar defaultValue={dayjs('2022-04-10')} onChange={onChange} />);
+      render(<DateCalendar defaultValue={dayjs('2022-04-21')} onChange={onChange} />);
```

Comment:
If we are aligning everywhere, then maybe going with the following would be a bit more correct? 🤔 
```suggestion
      render(<DateCalendar defaultValue={dayjs('2022-04-17')} onChange={onChange} />);
```